src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -1203,6 +1209,20 @@ private void AssertSnmp(ServerStore serverStore, LicenseStatus licenseStatus)
             throw new LicenseLimitException(LimitType.Snmp, message);
     }
 
+    private void AssertRemoteAttachmentsConfiguration(DatabaseRecord databaseRecord, LicenseStatus licenseStatus, ClusterOperationContext context)
+    {
+        if (licenseStatus.HasRemoteAttachments)
+            return;
+
+        if (databaseRecord.RemoteAttachments == null || databaseRecord.RemoteAttachments.HasDestination())
+            return;
+
+        if (CanAssertLicenseLimits(context, minBuildVersion: MinBuildVersion72000) == false)
```

Comment:
This isn't needed.
We add this only when we add a license limitation to an existing feature after it was released.